scripts/run_training.py

```diff
@@ -1,13 +1,12 @@
-import logging
-
 from yairl.util.trainer import init_trainer
 import gin.tf
 import argparse
+import tensorflow as tf
 
 
 @gin.configurable
 def main(env_name):
-    logging.getLogger().setLevel(logging.INFO)
+    tf.logging.set_verbosity(tf.logging.INFO)
```

Comment:
What are the advantages of using tf.logging relative to logging?

Guessing that by default it has a prettier print format than the root logger we've been using here

Reply:
I just thought that since we're using tf we should use its logging too.